projects/showcase/src/app/components/add-remove-list/showcase-add-remove-list.component.ts

```diff
@@ -18,6 +19,7 @@ export class ShowcaseAddRemoveListComponent {
 	private populateList(): void {
 		this.dataListButtonsRightSide.push(...this.generateDataList('ButtonsRightSide'));
 		this.dataListButtonsBottomSide.push(...this.generateDataList('ButtonsBottomSide'));
+		this.dataListButtonsCheckbox.push(...this.generateDataList('ButtonsBottomSide'));
```

Comment:
Should be ButtonsCheckbox instead of ButtonsBottomSide?

Reply:
Ok, fixed, thank you